chapters/Chapter 2 — Dressara.md

```diff
@@ -0,0 +1,94 @@
+
+But if any Dressaran woman—or, for that matter, any Dressaran at all—were ever to do anything interesting, Leanne would bet it would be Delphine. 
+
+Leanne looked away as Delphine approached. Her neck muscles tensed, her back straightened, her eyes glanced down—
+
+She jumped as a hand tapped her bare shoulder.
+
+“My apologies,” said a deep, melodic voice. “I didn’t mean to startle you.”
+
+It was Delphine. Leanne tore her eyes from her lips—halfway between smile and smirk, just as Leanne had seen before, and there was no mistaking that dark lipstick—up to Delphine’s eyes, kind and stalwart, framed by her short black hair.
+
+“A red,” Delphine instructed. “Merlot.”
+
+Leanne nodded readily. Her heat beat unevenly as she felt herself move towards the back of the garden, where the bar was.
+
+She had considered inviting herself to the garden party. She’d done as much before: the security at Dressaran parties was nonexistent: Dressarans tended to assume they were safe at all times. If they didn’t recognize her, they’d assume their Alt was forgetful. They’d make smalltalk just fine, and be friendly enough.
+
+But while Dressarans rarely had much of substance to say, when they did, they generally only shared it with those they knew. Even those that recognized Leanne from her previous intrusions would not feel so comfortable around her.
+
+Servants, however, were generally invisible. Their clothing depended on the party; today, they wore shimmering gray tunics held up by silvery strings that snaked over their shoulders.  
+
+Leanne shivered as she poured the wine. She was unused to exposing so much skin to the elements. The artificial wind tickled the scar upon her shoulder, the scar Delphine had tapped, the scar Leanne had received on the moon Delphinia after thirty seven assassins had cornered her. 
+
+She assumed Delphine was named after Delphinia. As much as the Dressarans looked down upon the other planets and moons, they frequently named their children after them: “Delphine,” “Delphy,” “Elise,” “Elsa.”
+
+Elsa was Delphine, Leanne assumed. She looked different now, of course. Her face had been soft, now it was angular. Her hair long and red, now short and black. But there was no mistaking her.
+
+She hadn’t been hard to track down. She’d only changed Face once. Faces records could only be unsealed with a half-dozen odd warrants, but Leanne didn’t need the records: while Leanne didn’t think she was all that great at tracking people down through their identity changes, she had to acknowledge that she was far better at it than any of her peers. She often hardly needed to research; she’d simply guess what people would do with their new identities, and frequently, she’d be right. 
+
+Leanne had guessed a Dressaran woman like Delphine wouldn’t miss out on the lavish parties her relations threw, even if she would now have to attend as a guest in order not to be associated with her former self and her former scandals. She had been right.
```

Comment:
It was unlikely Delphine knew anything of use. She was Dressaran, after all. Still, if Leanne was lucky, she could learn something useful from Delphine, and maybe she’d have some fun, too— not necessarily in that order.